File: evenodd.c
#include<stdio.h>
int main()
{
    int n;
    printf("enter the number :");
    scanf("%d",&n);
    if(n%2==0)
    {
        printf("even number");
    }
    else{
     printf("odd number :");
    }
    return 0;
}
void evenodd(int n)
{
       evenodd(n);
}

File: evenodd.java
import java.util.Scanner;
class evenodd
{
    public static void main(String args[])
    {
        int n;
        System.out.print("Enter the number :");
        Scanner r=new Scanner(System.in);
        n=r.nextInt();
        if(n%2==0)
        {
            System.out.print("Even");
        }
        else{
            System.out.print("odd");
        }
    }
}
